src/pages/vendor/VendorLayout/VendorSideBar/index.tsx

```diff
@@ -63,229 +60,255 @@ function VendorSideBar() {
+                        }}
+                      >
+                        <OrdersIcon />
+                      </ListItemIcon>
+                      <Typography
+                        variant="subtitle1"
+                        sx={{ marginLeft: '12px', fontSize: '16px' }}
+                      >
+                        {t('vendorSidebar.orders')}
+                      </Typography>
+                    </>
+                  ) : (
+                    <>
+                      <ListItemIcon
+                        sx={{
+                          minWidth: '24px',
+                          fill: theme.palette.text.disabled,
+                        }}
+                      >
+                        <OrdersIcon />
+                      </ListItemIcon>
+                      <Typography
+                        sx={{ marginLeft: '12px', fontWeight: '500' }}
+                      >
+                        {t('vendorSidebar.orders')}
+                      </Typography>
+                    </>
+                  )}
+                </Box>
+              </StyledListItemButton>
+            )}
+          </NavLink>
 
-          <StyledListItemButton
-            selected={selectedIndex === listIndexes.orders}
-            onClick={(event) => {
-              handleListItemClick(event, listIndexes.orders);
-              navigate(urls.VENDOR_ORDERS);
-            }}
-          >
-            <Box
-              display="flex"
-              justifyContent="flex-start"
-              alignItems="center"
-              width="140px"
-            >
-              {selectedIndex === listIndexes.orders ? (
-                <>
-                  <ListItemIcon
-                    sx={{ minWidth: '24px', fill: theme.palette.text.primary }}
-                  >
-                    <OrdersIcon />
-                  </ListItemIcon>
-                  <Typography
-                    variant="subtitle1"
-                    sx={{ marginLeft: '12px', fontSize: '16px' }}
-                  >
-                    {t('vendorSidebar.orders')}
-                  </Typography>
-                </>
-              ) : (
-                <>
-                  <ListItemIcon
-                    sx={{ minWidth: '24px', fill: theme.palette.text.disabled }}
-                  >
-                    <OrdersIcon />
-                  </ListItemIcon>
-                  <Typography sx={{ marginLeft: '12px', fontWeight: '500' }}>
-                    {t('vendorSidebar.orders')}
-                  </Typography>
-                </>
-              )}
-            </Box>
-          </StyledListItemButton>
+          <NavLink to={urls.VENDOR_CHATS}>
+            {({ isActive }) => (
+              <StyledListItemButton selected={isActive}>
+                <Box
+                  display="flex"
+                  justifyContent="flex-start"
+                  alignItems="center"
+                  width="140px"
+                >
+                  {isActive ? (
```

Comment:
find the way to do it with binary operation, its over code to add empty <> each tyme

Reply:
fixed unreadChatMessages